Menu/GameShell/10_Settings/Wifi/textarea.py

```diff
@@ -68,32 +71,71 @@ def AppendAndBlitText(self,alphabet):
         else:
             print("is Full %s" % "".join(self._MyWords))
 
+    def BuildBlitText(self):
+        blit_rows = [[]]
+        w = 0
+        xmargin = 5
+        endmargin = 15
+        x = self._PosX+xmargin
+        linenumber = 0
+        cursor_row = 0
+        
+        for i, v in enumerate(self._MyWords):
+            t = self._FontObj.render(v, True, (8, 135, 174))
+            t_width = t.get_width()
+            w += t_width
+            del(t)
```

Comment:
In hindsight, this may not be necessary.
I'm pretty sure python automatically will clean these up.